project/data_preparation/dataloader_service.py

```diff
@@ -24,18 +24,22 @@ def __call__(self, img):
         return torch.clamp(img, min=self.min_value, max=self.max_value)
 
 
-def get_dataset(dataset_name='Flowers'):
-    transforms = TF.Compose(
-        [
+def get_dataset(dataset_name='Flowers', , augment=False):
```

Comment:
Ez így szerintem nem futna 🏃 
```suggestion
def get_dataset(dataset_name='Flowers', augment=False):
```